packages/wdio-cucumber-framework/src/cucumberEventListener.js

```diff
@@ -0,0 +1,205 @@
+import { EventEmitter } from 'events'
+
+import { compareScenarioLineWithSourceLine, getStepFromFeature } from './utils'
+
+export default class CucumberEventListener extends EventEmitter {
+    gherkinDocEvents = []
+    acceptedPickles = []
+    currentPickle = null
+    testCasePreparedEvents = []
+
+    constructor (eventBroadcaster) {
+        super()
+        // attachEventLogger(eventBroadcaster)
+
+        eventBroadcaster
+            .on('gherkin-document', this.onGherkinDocument.bind(this))
+            .on('pickle-accepted', this.onPickleAccepted.bind(this))
+            .on('test-case-prepared', this.onTestCasePrepared.bind(this))
+            .on('test-case-started', this.onTestCaseStarted.bind(this))
+            .on('test-step-started', this.onTestStepStarted.bind(this))
+            .on('test-step-finished', this.onTestStepFinished.bind(this))
+            .on('test-case-finished', this.onTestCaseFinished.bind(this))
+            .on('test-run-finished', this.onTestRunFinished.bind(this))
+    }
+
+    // gherkinDocEvent = {
```

Comment:
Is there a better way to store event examples? They are definitely useful, just thinking on some more reliable / proper way.

Reply:
these are comments, not sure how to make them more reliable. Given there is the only position you would be interested in this, I would keep them.